executor/prepared_test.go

```diff
@@ -113,6 +113,27 @@ func (s *testSuite) TestPrepared(c *C) {
 		_, err = tk.Exec("execute stmt")
 		c.Assert(err, NotNil)
 
+		_, _, fields, err := tk.Se.PrepareStmt("select a from prepare3")
```

Comment:
Could you add a test case with placeholder? And also a test case of update statement.